src/declarations.d.ts

```diff
@@ -0,0 +1,3 @@
+// TODO: These can be removed once these modules have their own type definitions
+declare module 'resin-sdk';
+declare module 'resin-settings-client';
```

Comment:
Can you add the .editorconfig to this project and ensure newlines at the end of the files?
https://github.com/resin-io-modules/resin-analytics/blob/8004cb482e1ddebeb94ff0bf6a836f3a7d061d4a/packages/resin-mixpanel-client/.editorconfig is a standard one we use in some other modules.